bin/buildBlueprints.js

```diff
@@ -52,7 +52,7 @@ function loadBlueprintsFromPath(filePath, isProduction) {
     // more useful than the extensions object, and easier to understand. I'd
     // like to deprecate the extensions object if its not being used in many places.
     if (typeof builds === 'function') {
-      builds = builds(isProduction);
+      builds = builds(options);
```

Comment:
this gives more flexibility to the person writing the config to do what they need based on the env / targets.